### PR TITLE
OCPBUGS-37699: Remove mode=6 balance-alb from networking docs

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.12+

Issue:
[OCPBUGS-37699](https://issues.redhat.com/browse/OCPBUGS-37699)

Link to docs preview:
[Example: Bond interface node network configuration policy](https://79684--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.html#virt-example-bond-nncp_k8s_nmstate-updating-node-network-config)

- [x] SME has approved this change.
- [x] QE has approved this change.

### CODE REVIEW

File: modules/virt-example-bond-nncp.adoc [hunk 1]
```diff
@@ -15,8 +15,6 @@ to the cluster.
 * mode=1 active-backup +
```
Comment (dfitzmau) [inline modules/virt-example-bond-nncp.adoc:15]: Article https://access.redhat.com/solutions/67546 mentions that mode 1 is partially working..do I need to call out what might happen on using this mode?

**Comment by qiowang721** [comment]
Hi @dfitzmau could you also help to update the bond mode in this example? It uses balance-rr, which is also not in the supported list: https://docs.openshift.com/container-platform/4.15/networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.html#virt-example-nmstate-multiple-interfaces_k8s_nmstate-updating-node-network-config

**Comment by cybertron** [comment]
/lgtm

It might be helpful to also mention supported bond modes in the day 1 docs: https://docs.openshift.com/container-platform/4.16/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-host-dual-network-interfaces-in-the-install-config-yaml-file_ipi-install-installation-workflow

Right now it's more important to pick the right mode at install time than in an NNCP since the NNCP can't apply to the primary interface anyway.

File: modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc [hunk 2]
```diff
@@ -6,10 +6,16 @@
...
 
+Before installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState to support dual port NIC.
+
 :FeatureName: Support for Day 1 operations associated with enabling NIC partitioning for SR-IOV devices
 include::snippets/technology-preview.adoc[leveloffset=+1]
 
-Before installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState to support dual port NIC.
+{VirtProductName} only supports the following bond modes:
+
+* mode=1 active-backup +
```
Comment (kcarmichael08) [inline modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc:16]: Looks like from the config example that the mode is actually just the `active-backup` part, for example, so should there be some kind of punctuation after the "mode=x" part? 

Example:

* mode=1: active-backup

(I'm not familiar with this subject so maybe not, just a suggestion!)

**Review by kcarmichael08 — COMMENTED** [review]
one question; lgtm!

File: modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc (hunk 2 above)
Comment (dfitzmau) [reply modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc:16]: Confirmed with the SME that the pluses are not related to the bond names but instead are backend Asciidoc rendering that might be needed as an `=` and a `:` are used.

**Review by sheriff-rh — APPROVED** [review]
Changes LGTM; merging.

**Comment by sheriff-rh** [comment]
Looks like a manual cherrypick for 4.12 @dfitzmau

**Pull request merged**